tests/Doctrine/Tests/DBAL/Functional/StatementTest.php

```diff
@@ -50,6 +50,60 @@ public function testStatementIsReusableAfterClosingCursor(): void
         self::assertEquals(2, $id);
     }
 
+    public function testFetchAllWithOneArgument(): void
+    {
+        $table = new Table('stmt_fetch_all');
```

Comment:
You don't have to create a temporary table just to fetch one row. You can try using `TestUtil::generateResultSetQuery()` instead.

Reply:
Where is this method located? I can't find it anywhere!

Reply:
Ah, sorry. It was only introduced in `2.11.x` as part of https://github.com/doctrine/dbal/pull/4019. Feel free to leave your test as is or backport it.